sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java

```diff
@@ -134,6 +135,16 @@ public void run() {
         }
     }
 
+
+    public void runJobContextReapers() throws Exception{
+        Class<?> innerClass = Class.forName("io.crate.jobs.JobContextService$Reaper");
```

Comment:
i would like to use the Reaper, so it's ensured that these context would be closed by the reaper.